pom.xml

```diff
@@ -0,0 +1,202 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ * Copyright (c) 2023 General Motors GTO LLC.
```

Comment:
Copyright information needs to be corrected (this is not from GM and not in 2023)

Reply:
removed